CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/),
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased] - yyyy-mm-dd
+### Added
+- `datepicker` Added documentation.
```

Comment:
This should not be added to the changelog

Reply:
Is this not consistent with the previous changelog update, where added documentation is mentioned?

Reply:
My bad, I thought this was the component's changelog. You can ignore my comment 😇